test/integration/authorization_test.go

```diff
@@ -1172,6 +1173,19 @@ func TestAuthorizationSubjectAccessReview(t *testing.T) {
 	}.run(t)
 }
 
+// used for doing raw REST
+func createNewProjectRaw(clusterAdminClient client.Interface, clientConfig restclient.Config, projectName, adminUser string) (*client.Client, error) {
+	adminClient, err := testserver.CreateNewProject(clusterAdminClient, clientConfig, projectName, adminUser)
+	if err != nil {
+		return nil, err
+	}
+	rawClient, ok := adminClient.(*client.Client)
```

Comment:
@liggitt This is the only type cast I was not able to remove without adding a bunch of code.